rust-bindings/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+mod bindings {
+    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
```

Comment:
is this standard practice for using bindgen? It looks shady to depend on an environment variable at build time.

Reply:
Yes

https://rust-lang.github.io/rust-bindgen/tutorial-4.html
https://doc.rust-lang.org/cargo/reference/environment-variables.html
> `OUT_DIR` If the package has a build script, this is set to the folder where the build script should place its output. See below for more information. (Only set during compilation.)